projects/addon-editor/components/edit-link/utils/edit-link-parse-url.ts

```diff
@@ -0,0 +1,63 @@
+import {
+    TUI_EDITOR_LINK_HASH_PREFIX,
+    TUI_EDITOR_LINK_OSI_PROTOCOL_DIVIDER,
+    TUI_EDITOR_LINK_SIMPLE_PROTOCOL_DIVIDER,
+} from '@taiga-ui/addon-editor/constants';
+import {tuiIsValidUrl} from '@taiga-ui/cdk';
+
+interface TuiEditLinkParsed {
+    prefix: string;
+    path: string;
+}
+
+export function tuiEditLinkParseUrl(url?: string): TuiEditLinkParsed {
+    if (url?.startsWith(TUI_EDITOR_LINK_HASH_PREFIX)) {
+        const [anchor = ``] = url?.split(`#`).slice(-1) ?? [];
```

Comment:
Why not `TUI_EDITOR_LINK_HASH_PREFIX`? Also if it starts with it, why do we need to split and not just use `.slice(1)`?